src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -28,18 +31,18 @@ import {
   AccordionItemButton
 } from 'src/shared/components/accordion';
 import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
-
-import { getEntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
+import { defaultSort } from 'src/content/app/entity-viewer/shared/helpers/transcripts-sorter';
```

Comment:
Can this be moved above next to non-components imports?

Reply:
✔️